gpjax/base/module.py

```diff
@@ -202,6 +200,16 @@ def _apply_stop_grad(meta_leaf):
 
         return meta_map(_apply_stop_grad, self)
 
+    def trainables(self) -> Self:
```

Comment:
```suggestion
    def trainables(self: Self) -> Self:
```
otherwise "Self" doesn't contain any information